ReactWindows/ReactNative/Collections/ListExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
```

Comment:
Clean up unused namespaces here. (In Visual Studio, right-click, remove and sort namespaces).